api/blocks/blocks.go

```diff
@@ -95,5 +95,6 @@ func (b *Blocks) Mount(root *mux.Router, pathPrefix string) {
 	sub := root.PathPrefix(pathPrefix).Subrouter()
 	sub.Path("/{revision}").
 		Methods(http.MethodGet).
-		HandlerFunc(utils.MetricsWrapHandlerFunc(pathPrefix, "blocks_get_block", b.handleGetBlock))
+		Name("blocks_get_detail").
```

Comment:
Would be good to keep the name as close as possible to the URL and avoid the interpretation step.
Ie. `curl .../block/0x0000` -> `blocks_get_block`